EventListener/ReplaceImageListener.php

```diff
@@ -0,0 +1,126 @@
+            sprintf('<img alt="Exception detected!" data-gif src="%s" style="width:145px" />', $this->generateUrl(sprintf('bundles/gifexception/images/%s/%s', $dir, $gif))),
```

Comment:
Good catch. In first place, I only matched "src" part, but now that I also match the "alt" attribute, we can safely use a regex pattern for the "src" :wink: